src/content/self-hosted/install/releases.mdx

```diff
@@ -328,229 +328,4 @@ January 19, 2023
 - Fixed blank screen that sometimes appeared when clicking the "Stop development" button
 - Fixed race condition when trying to deploy a development environment that was removed from the catalog
 - Ingresses that are not managed for Okteto won't be deleted when a namespace is put to sleep
-- Fixed how host aliases are injected in the installer pods as the connection to Okteto's API and registry was not using the internal IPs in some scenarios
```

Comment:
@rberrelleza Would it make sense to keep these older release notes for posterity? I don't feel strongly that we need to keep them, but it could be helpful for context gathering for new hires.

Maybe at a minimum we could move these to Notion?